hushline/model.py

```diff
@@ -28,6 +28,14 @@ def default(cls) -> "SMTPEncryption":
         return cls.StartTLS
 
 
+class HostOrganization(Model):
+    __tablename__ = "host_organization"
+
+    id: Mapped[int] = mapped_column(primary_key=True, default=1)
+    brand_app_name: Mapped[str] = mapped_column(db.String(255), default="🤫 Hush Line")
+    brand_primary_hex_color: Mapped[str] = mapped_column(db.String(255), default="#7d25c1")
```

Comment:
should we limit the length to 7?

Reply:
Addressed by (https://github.com/scidsg/hushline/pull/573/commits/ab0a21251f9a94858515c724e86c00f3f172ec27)